### PR TITLE
remove support for string arguments to start and mark the argument as…

### PR DESCRIPTION
**Pull request by mikebolt**
… optional

### CODE REVIEW

**Comment by mikebolt** [comment]
see #417

File: src/Tween.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export class Tween<T extends UnknownProps> {
 		return this
 	}
 
-	start(time: number): this {
+	start(time?: number): this {
```
Comment (trusktr) [inline src/Tween.ts:73]: :ok

File: src/Tween.ts [hunk 2]
```diff
@@ -101,8 +101,13 @@ export class Tween<T extends UnknownProps> {
 
 		this._isChainStopped = false
 
-		this._startTime =
-			time !== undefined ? (typeof time === 'string' ? TWEEN.now() + parseFloat(time) : time) : TWEEN.now()
+		if (time === undefined) {
+			this._startTime = TWEEN.now()
+		}
+		else {
+			this._startTime = time
```
Comment (trusktr) [inline src/Tween.ts:108]: What about `TWEEN.now() + time`?

**Comment by trusktr** [comment]
`npm run lint` will fix the formatting and the tests will pass. But I'm curious if removing the `now()+time` is a breaking change.

File: src/Tween.ts (hunk 2 above)
Comment (trusktr) [reply src/Tween.ts:108]: On second though, passing a string looks totally broken here. I'll fix this up and remove.

**Pull request closed without merging**